### DISCUSSION TITLE
Is createFile/writeFile/appendFile possible in MQTTX script ?

### DISCUSSION THREAD

**Discussion by meuron-sk** [emqx/MQTTX#684] [Q&A]
Want to filter and write/append messages to local file.
Is createFile/writeFile/appendFile possible in MQTTX script ??

**Comment by oceanlvr** [comment]
Sorry for delay replying, it's a great suggestion that support IO operation for script feature.

**Comment by ysfscream** [comment]
This is indeed a good suggestion, but I think it still needs to be discussed. The script is run in a sandbox, I don't know if there would be permissions or security issues if operating on files, maybe we could implement a file saving feature for messages?
